lib/foreman/cast.rb

```diff
@@ -0,0 +1,22 @@
+module Foreman
+  module Cast
+    def self.to_bool(value)
+      case value
+
+      when String
+        return true if value =~ (/^(true|t|yes|y|1)$/i)
```

Comment:
use \A and \z to start and terminate these regexes, as ^ and $ match lines, not strings